proxysql/build-binary-proxysql

```diff
@@ -9,6 +9,16 @@
 set -o errexit
 set -o xtrace
 
+if [ -f /usr/bin/yum ]; then
+    RHEL=$(rpm --eval %rhel)
+    ARCH=$(echo $(uname -m) | sed -e 's:i686:i386:g')
+    if [[ $RHEL -eq 8 ]]; then
+        sudo yum -y install python2 gnutls-devel libtool || true
+    fi
+    sudo yum -y install wget
+    sudo yum -y install automake bzip2 cmake make gcc-c++ gcc git openssl openssl-devel gnutls gnutls-devel libtool patch
+fi
+
```

Comment:
From CentOS 8 logs I see these dependencies have already been there https://pxc.cd.percona.com/job/qa-proxysql2-pipeline/303/consoleFull

```
Package python2-2.7.17-1.module_el8.2.0+381+9a5b3c3b.x86_64 is already installed.
Package gnutls-devel-3.6.8-11.el8_2.x86_64 is already installed.
Package libtool-2.4.6-25.el8.x86_64 is already installed.
Dependencies resolved.
Nothing to do.

+ sudo yum -y install wget
Last metadata expiration check: 0:00:02 ago on Wed Nov 25 09:52:11 2020.
Package wget-1.19.5-8.el8_1.1.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!

+ sudo yum -y install automake bzip2 cmake make gcc-c++ gcc git openssl openssl-devel gnutls gnutls-devel libtool patch
Last metadata expiration check: 0:00:03 ago on Wed Nov 25 09:52:11 2020.
Package automake-1.16.1-6.el8.noarch is already installed.
Package bzip2-1.0.6-26.el8.x86_64 is already installed.
Package cmake-3.11.4-7.el8.x86_64 is already installed.
Package make-1:4.2.1-10.el8.x86_64 is already installed.
Package gcc-c++-8.3.1-5.el8.0.2.x86_64 is already installed.
Package gcc-8.3.1-5.el8.0.2.x86_64 is already installed.
Package git-2.18.4-2.el8_2.x86_64 is already installed.
Package openssl-1:1.1.1c-15.el8.x86_64 is already installed.
Package openssl-devel-1:1.1.1c-15.el8.x86_64 is already installed.
Package gnutls-3.6.8-11.el8_2.x86_64 is already installed.
Package gnutls-devel-3.6.8-11.el8_2.x86_64 is already installed.
Package libtool-2.4.6-25.el8.x86_64 is already installed.
Package patch-2.7.6-11.el8.x86_64 is already installed.
Dependencies resolved.
```

CentOS 7 only updates the current installed deps and installs cmake from percona-dev repository
```
 cmake               x86_64     3.2.2-1.el7.centos        percona-dev     7.0 M
```